pkgs/applications/science/electronics/kicad/libraries.nix

```diff
@@ -42,30 +28,4 @@ in
   templates = mkLib "templates";
   footprints = mkLib "footprints";
   packages3d = mkLib "packages3d";
-
-  # i18n is a special case, not actually a library
```

Comment:
i think this description of i18n being optional, separate and only found via a relative path should have been retained